internal/models/tokens.go

```diff
@@ -31,3 +31,17 @@ type CustomClaims struct {
 	// https://datatracker.ietf.org/doc/html/rfc7519#section-4.1
 	jwt.RegisteredClaims
 }
+
+// JSONResponse defines a JSON response struct.
+type JSONResponse struct {
+	AccessToken      string `json:"access_token,omitempty"`
+	Error            string `json:"error,omitempty"`
+	ErrorDescription string `json:"error_description,omitempty"`
+}
```

Comment:
ok, no.

You have an `ErrorResponse` and then, you have the other model representing whatever you have to represent.

what is error and errordescription?

Reply:
hmm, ok.
follow some examples:
```
{"error":"Bad Request","error_description":"an unexpected error occurred while validating the JWT token"}
```
```
{"error":"Bad Request","error_description":"that's not even a token: Barear eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJuYW1lIjoiV2FsbGFjZSBTYWxsZXMiLCJlbWFpbCI6I2VydmljZSIsImV4cCI6MTYzOTYwOTAyMSwiaWF0IjoxNjM5NTIyNjIxfQ.2FsiJ16qDeTih_hRIOgVADk7qJUiKVvzjBt76ucyryY"}
```

Reply:
please, see again and if you agree.

Reply:
Humm, for erros we can use a specific struct. Something like that:
```golang
type ErrorResponse struct {
   Error            string
   Message     string
   Timestamp time.Time
} 
```
```json
{
    "timestamp":"2019-09-16T22:14:45.624+0000",    
    "error":"Internal Server Error",
    "message":"No message available",
}
```
And other struct for the tokens, for example in the future we can have a refresh_token as well.

Reply:
now we've only the message field.

Reply:
I added expiresAt (success) and timestamp (error). 👍